deepfence_agent/tools/apache/scope/backend/Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM golang:1.16.2
+RUN wget https://go.dev/dl/go1.19.2.linux-amd64.tar.gz
```

Comment:
Can we not directly use `golang:1.19.2` in the above `FROM` statement?

Reply:
`golang:1.19` is lacking some of the lib/bin we use afterwards

Reply:
`golang:1.19.2-buster` worked great instead :)